### DISCUSSION TITLE
DataTable dynamic column exporting

### DISCUSSION THREAD

**Discussion by mariusurbelis** [primefaces/community#284] [PrimeVue]
Moving the question from the forum here:

https://forum.primefaces.org/viewtopic.php?f=110&t=81861

**Comment by tugcekucukoglu** [comment]
Could you update the codesandbox demo? An error occurs `[Vue warn]: Invalid prop: type check failed for prop "field". Expected String | Function, got Number with value 1.`
The issue could be related to this.

**Reply by mariusurbelis** [reply]
@tugcekucukoglu thanks for spotting that. I've changed ids to string type and fixed this warning. It was not related.
I get this type of export without the data from dynamic columns:
<img width="1230" alt="image" src="https://github.com/primefaces/community/assets/32742633/fcc973c4-c6d8-4e85-9e8a-9a348b37c071">
